manifests/b/Bloodstrike/pc/1.1/Bloodstrike.pc.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Bloodstrike.pc

```

```suggestion
PackageIdentifier: NetEase.BloodStrike
```
- The game seems to capitalize and separate the second word in the name `Blood Strike`, so the `PackageIdentifier` should have this capitalized too. 
- Since `NetEase Games` is the developer, should this be added to the `PackageIdentifier`? If not, then the current `PackageIdentifier` is okay. 
- Please remember to modify the file name and path to reflect any changes to the `PackageIdentifier`.